### ISSUE TITLE
Write tests for service multi_media_pick_service.dart

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#2162]
The Talawa code base needs to be 100% reliable. This means we need to have 100% unittest code coverage.

We will need unittests done for all methods, classes and functions found in this file.

Any widgets referenced in this file must also have unittests done

```
lib/services/third_party_service/multi_media_pick_service.dart
```

### PR Acceptance Criteria

- When complete this file must show **100%** coverage when merged into the code base.
- The current code coverage for this file can be found here:
    - https://app.codecov.io/gh/PalisadoesFoundation/talawa?displayType=list
- The PR will show a report for the code coverage for the file you have added. You can use that as a guide.
- No functionality must be removed to achieve this goal.

**Comment by shahzeelahmed** [comment]
@noman2002 could you please assign this issue to me

**Comment by Azad99-9** [comment]
@Cioppolo14 please assign this issue to me.

**Comment by Azad99-9** [comment]
@palisadoes @noman2002 please close this issue as it is fixed in #2265 PR.
